arelle/plugin/validate/NL/rules/fr_nl.py

```diff
@@ -21,6 +22,38 @@
 
 _: TypeGetText
 
+@validation(
+    hook=ValidationHook.XBRL_FINALLY,
+    disclosureSystems=[
+        DISCLOSURE_SYSTEM_NT16,
+        DISCLOSURE_SYSTEM_NT17,
+        DISCLOSURE_SYSTEM_NT18,
+    ],
+)
+def rule_fr_nl_1_06(
+        pluginData: PluginValidationDataExtension,
+        val: ValidateXbrl,
+        *args: Any,
+        **kwargs: Any,
+) -> Iterable[Validation] | None:
+    """
+    FR-NL-1.06: The file name of an XBRL instance document MUST NOT contain characters with different meanings on different platforms.
+    Only characters [0-9], [az], [AZ], [-] and [_] (File names not including the extension and separator [.]).
+    """
+    pattern = re.compile(r"^[\w-]*$")
```

Comment:
I think `\w` is Unicode-aware, but I don't think we want that:
```
>>> re.compile(r"^[\w-]*$").match("ööps")
<re.Match object; span=(0, 4), match='ööps'>
```
Suggest `[0-9a-zA-Z_-]`.